uncontrast_studies/models/consciousness_measure.py

```diff
@@ -0,0 +1,74 @@
+from django.db import models
+from django.db.models import CASCADE
+
+
+class UnConsciousnessMeasurePhase(models.Model):
+    name = models.CharField(blank=False, null=False, max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class UnConsciousnessMeasureType(models.Model):
+    name = models.CharField(blank=False, null=False, max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class UnConsciousnessMeasureSubType(models.Model):
+    name = models.CharField(blank=False, null=False, max_length=30)
+    type = models.ForeignKey(
+        blank=False,
+        null=False,
+        on_delete=CASCADE,
+        to=UnConsciousnessMeasureType,
+        related_name="unconsciousness_measure_sub_types",
+    )
+
+    def __str__(self):
+        return self.name
+
+
+class UnConsciousnessMeasure(models.Model):
+    experiment = models.ForeignKey(
+        to="uncontrast_studies.UnConExperiment", on_delete=CASCADE, related_name="unconsciousness_measures"
+    )
+    phase = models.ForeignKey(
+        blank=False,
+        null=False,
+        on_delete=CASCADE,
+        to=UnConsciousnessMeasurePhase,
+        related_name="unconsciousness_measures",
+    )
+    type = models.ForeignKey(
+        blank=False,
+        null=False,
+        on_delete=CASCADE,
+        to=UnConsciousnessMeasureType,
+        related_name="unconsciousness_measures",
+    )
+    sub_type = models.ForeignKey(
+        blank=False,
+        null=False,
+        on_delete=CASCADE,
+        to=UnConsciousnessMeasureSubType,
+        related_name="unconsciousness_measures",
+    )
+    # Consciousness Measure metadata
+    number_of_trials = models.PositiveSmallIntegerField(
+        null=True, blank=True, verbose_name="consciousness measure # of trials for the objective measure"
+    )
+    number_of_participants_in_awareness_test = models.PositiveSmallIntegerField(null=True, blank=True)
+    is_cm_same_participants_as_task = models.BooleanField(
+        null=False,
+        blank=False,
+        verbose_name="is consciousness measure taken from the same participants as the main task",
+    )
+    is_performance_above_chance = models.BooleanField(null=False, blank=False)
+    is_trial_excluded_based_on_measure = models.BooleanField(
```

Comment:
no default here? (also above) you prefer explicit input?

Reply:
זאת לא העדפה per se, פשוט לא ידעתי את התשובה. שואלת את הצוות שלהם בסלאק.

Reply:
they insist on no default